src/app/destiny2/d2-definitions.ts

```diff
@@ -179,6 +179,9 @@ export function buildDefinitionsFromManifest(db: AllDestinyManifestComponents) {
         if (!dbTable) {
           throw new Error(`Table ${table} does not exist in the manifest`);
         }
+        if (id < 1 || id > Number.MAX_SAFE_INTEGER) {
+          throw new Error(`Hash ${id} does not exist in the manifest`);
```

Comment:
think this could be clearer. 9999999 also might "not exist in manifest". this is more of an "invalid hash"

Reply:
true, will fix